src/lib/probe.c

```diff
@@ -87,6 +175,14 @@ CbmDeviceProbe *cbm_probe_path(const char *path)
                 }
         }
 
+        /* Ensure that this parent disk really is LUKS */
+        if (blkid_probe_lookup_value(blk_probe, "TYPE", &value, NULL) != 0) {
```

Comment:
This workflow is getting lost to me but if you goto clean then ret is going to be NULL. I'd think you'd only want to have the probe.luks_uuid unset in this case though instead of throwing out all the probe values?